app/src/main/java/com/keylesspalace/tusky/ViewThreadActivity.java

```diff
@@ -35,9 +35,17 @@
 
 public class ViewThreadActivity extends BaseActivity implements HasSupportFragmentInjector {
 
+    public static final int REVEAL_BUTTON_HIDDEN = 1;
+    public static final int REVEAL_BUTTON_REVEAL = 2;
+    public static final int REVEAL_BUTTON_HIDE = 3;
```

Comment:
Why not an enum?

Reply:
Hard question.
Probably because it's such a small feature and it's pretty much local I didn't want to add an overhead of a class.